pkg/ai/ai.go

```diff
@@ -87,3 +87,7 @@ func (a *OpenAIClient) Parse(ctx context.Context, prompt []string, nocache bool)
 	}
 	return response, nil
 }
+
+func (a *OpenAIClient) GetName() string {
+	return "fakeai"
```

Comment:
Shouldn't it be `openai` instead of `fakeai` here?

Reply:
+1 on this

Reply:
fixed this